spring/pom.xml

```diff
@@ -14,34 +14,24 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>transformation-distros-spring</artifactId>
-    <name>Transformation :: Service :: Distros :: Spring</name>
-
     <parent>
-        <artifactId>transformation-distros</artifactId>
         <groupId>com.connexta.transformation</groupId>
+        <artifactId>transformation-service</artifactId>
         <version>0.1.0-SNAPSHOT</version>
     </parent>
 
+    <artifactId>transformation-distros-spring</artifactId>
```

Comment:
✏️ Think we could remove the word "distros" from the artifactId now that we're removing the "distros" hierarchy?

Reply:
agreed. let's do that in the next PR. I didn't want to impact the artifact id right now since we are trying to get the build running